applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -75,7 +75,16 @@ def _create_solution_scheme(self):
             mechanical_scheme = KratosMultiphysics.ResidualBasedBossakDisplacementScheme(damp_factor_m)
         elif(scheme_type.startswith("bdf") or scheme_type == "backward_euler"):
             order = self._bdf_integration_order()
-            mechanical_scheme = KratosMultiphysics.ResidualBasedBDFDisplacementScheme(order)
+            # In case of rotation dof we declare the dynamic variables
```

Comment:
guys @KratosMultiphysics/structural-mechanics this files is becoming a monster file.

not as a part of this PR but we should discuss on how to divide it in several subfiles (i guess by derivation) to make it manageable

Reply:
It is not really really urgent, but @KratosMultiphysics/structural-mechanics asked me for this, in fact this more like PR discussion than a pure PR